core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -97,15 +97,14 @@ oppia.controller('FeedbackTab', [
     };
 
     var _isSuggestionHandled = function() {
```

Comment:
Move some (most?) of these methods to be on the SuggestionThread object. E.g. suggestionThread.isSuggestionHandled() etc. 

Add unit tests.